### PR TITLE
Button

### PR DESCRIPTION
**Pull request by Cindie1908**
add button in order to close card "ShowIngredient"

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -14,7 +14,10 @@ function Home () {
 				<div>
 					<Title />
 					<img className="séparateur" src="https://zupimages.net/up/19/44/l8sr.png" alt="séparateur"></img>
+<<<<<<< HEAD
+=======
 					{/* <img className="séparateur" src="http://meriamr.m.e.pic.centerblog.net/066bb9f4.gif" alt="séparateur"></img> */}
+>>>>>>> dev
```
Comment (erabin05) [inline src/components/Home.js]: you didn't took care of those conflict 1)

File: src/components/Home.js [hunk 2]
```diff
@@ -29,7 +32,12 @@ function Home () {
 				</div>
 				<div className="affichageBouton">
 					<h2>Sélectionner votre poison :</h2>
+<<<<<<< HEAD
+						<div className="HomeButton">
+
+=======
 					<div className="HomeButton">
+>>>>>>> dev
```
Comment (erabin05) [inline src/components/Home.js]: you didn't took care of those conflict 2)

**Review by erabin05 — CHANGES_REQUESTED** [review]
Just a file with elements of a conflict

**Comment by Cindie1908** [comment]
close with merge presentation

**Pull request closed without merging**